prod-sample/sample-modules/cultured-frankie/0.0.0/src/components/CulturedFrankie.jsx

```diff
@@ -53,6 +51,11 @@ CulturedFrankie.propTypes = {
   }).isRequired,
   localeName: PropTypes.string.isRequired,
 };
+CulturedFrankie.holocron = {
+  name: 'cultured-frankie',
+  load: () => (dispatch) => dispatch(loadLanguagePack('cultured-frankie', { fallbackLocale: 'en-US' })),
```

Comment:
`load` should probably be changed to `loadModuleData` as we are deprecating `load` 
https://github.com/americanexpress/holocron/blob/master/packages/holocron/docs/api/README.md#moduleholocron

Reply:
Good catch, I'll update that!

Reply:
Updated (0dda01b)